rsnapshot-program.pl

```diff
@@ -6716,6 +6739,28 @@ =head1 CONFIGURATION
 
 =back
 
+B<include_conf_dir>   Include any files in this directory at this point, defaults ending in .conf
+
+=over 4
+
+This is also recursive configuration so it's important to be careful about what snippets
```

Comment:
Why is it "snippets" now? They're just included configuration files.  Also suggest change to "... so its important included files don't cause an infinite loop".